src/lib/modules/Document/DocumentInfo.js

```diff
@@ -58,6 +70,12 @@ export class DocumentInfo extends Component {
               <Table.Cell width={4}>Title</Table.Cell>
               <Table.Cell>{metadata.title}</Table.Cell>
             </Table.Row>
+            <Table.Row>
+              <Table.Cell width={4}>Alternative titles</Table.Cell>
+              <Table.Cell>
+                {this.renderAlternativeTitles(metadata.alternative_titles)}
```

Comment:
```suggestion
                <List bulleted>
        {titles.map((entry) => (
          <List.Item key={entry.value}>
            <List.Content>{entry.value}</List.Content>
          </List.Item>
        ))}
      </List>
```
I would keep the JSX in the render method, especially because we are not gaining much by separating 5 lines of the code to new method

Reply:
fair!